StarEtc/agmlupd.csh

```diff
@@ -0,0 +1,11 @@
+#
```

Comment:
We can remove this shell script because  geometry build dependencies have been fixed.